google-cloud-debugger/README.md

```diff
@@ -1,32 +1,188 @@
+[Stackdriver Debugger](https://cloud.google.com/debugger/) lets you inspect the state of a running application at any code location in real time, without stopping or slowing down the application, and without modifying the code to add logging statements. You can use Stackdriver Debugger with any deployment of your application, including test, development, and production. The Ruby debugger adds minimal request latency, typically less than 50ms, and only when the application state is captured. In most cases, this is not noticeable by users.
 
 - [google-cloud-debugger documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-debugger/master/google/cloud/debugger)
 - [google-cloud-debugger on RubyGems](https://rubygems.org/gems/google-cloud-debugger)
 - [Stackdriver Debugger documentation](https://cloud.google.com/debugger/docs/)
 
 ## Quick Start
 
-Setting up Stackdriver Debugger involves three steps:
+Install the gem directly:
 
-1. Add the `google-cloud-debugger` library to your app.
-2. Register your app's source code.
-3. Deploy your app and set a breakpoint.
+```sh
+$ gem install google-cloud-debugger
+```
 
-See the
-[google-cloud-debugger documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-debugger/master/google/cloud/debugger)
-for a quick tutorial.
+Or install through Bundler:
 
-## Authentication
+1. Add the `google-cloud-debugger` gem to your Gemfile:
+
+```ruby
+gem "google-cloud-debugger"
+```
+
+2. Use Bundler to install the gem:
+
+```sh
+$ bundle install
+```
+
+Alternatively, check out the [`stackdriver`](../stackdriver) gem that includes 
+the `google-cloud-debugger` gem.
+
+## Enable Stackdriver Debugger API
+
+The Stackdriver Debugger agent needs the [Stackdriver Debugger 
+API](https://console.cloud.google.com/apis/library/clouddebugger.googleapis.com) 
+to be enabled on your Google Cloud project. Make sure it's enabled if not 
+already.
+
+## Enabling the Debugger agent
+
+The Stackdriver Debugger library provides a Debugger agent that helps instrument
+breakpoints in your running applications. The library also comes with a Railtie
+and a Rack Middleware to help control the Debugger agent in popular Rack based
+frameworks, such as Ruby on Rails and Sinatra.
+
+### With Ruby on Rails
+
+You can load the Railtie that comes with the library into your Ruby 
+on Rails application by explicitly require it in the application startup path:
```

Comment:
"explicitly requiring it during application startup"

Reply:
Updated